.github/workflows/build.yml

```
@@ -1,31 +1,86 @@
+          retention-days: 7
+          path: pack/*.nupkg
+        if: ${{ github.event_name == 'release'  && github.ref_type == 'tag' || github.event.release.tag_name }}
+
       - name: Publish .NET Solution to NuGet.org
         continue-on-error: true
-        run: dotnet nuget push pack/* --api-key ${{ secrets.NUGET_ORG_KEY }} --source nuget
+        env:
+          apikey: ${{ secrets.NUGET_ORG_KEY }}
+        run: dotnet nuget push pack/*.nupkg --api-key ${{ secrets.NUGET_ORG_KEY }} --source nuget
+        if: ${{ env.apikey != '' && github.event_name == 'release'  && github.ref_type == 'tag' || github.event.release.tag_name }}
         
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)